Makefile

```diff
@@ -114,13 +114,6 @@ integration-test:  ## Run integration test
 	@go test ./test
 .PHONY: integration-test
 
-check: boilerplate-check  ## Check code formats
```

Comment:
Actually this is used in https://github.com/dragonflyoss/Dragonfly/blob/9017881ccebde40b1ea47b875b71c4122b13333e/.circleci/config.yml#L59.

I do not think we could delete these.

Reply:
I just removed the `check`, so we cannot use `make check` anymore.
But we can still use `make boilerplate-check` in ci. This does not conflict with my change.

Reply:
I agree. My misunderstanding.